BrewMesModulesParent/machine/src/main/java/com/brewmes/machine/MachineServiceImpl.java

```diff
@@ -0,0 +1,259 @@
+package com.brewmes.machine;
+
+import com.brewmes.common.entities.Connection;
+import com.brewmes.common.services.IMachineService;
+import com.brewmes.common.util.Command;
+import com.brewmes.common.util.Products;
+import com.brewmes.common.util.machinenodes.CommandNodes;
+import com.brewmes.common_repository.ConnectionRepository;
+import org.eclipse.milo.opcua.sdk.client.OpcUaClient;
+import org.eclipse.milo.opcua.sdk.client.api.config.OpcUaClientConfigBuilder;
+import org.eclipse.milo.opcua.stack.client.DiscoveryClient;
+import org.eclipse.milo.opcua.stack.core.UaException;
+import org.eclipse.milo.opcua.stack.core.types.builtin.DataValue;
+import org.eclipse.milo.opcua.stack.core.types.builtin.NodeId;
+import org.eclipse.milo.opcua.stack.core.types.builtin.Variant;
+import org.eclipse.milo.opcua.stack.core.types.structured.EndpointDescription;
+import org.eclipse.milo.opcua.stack.core.util.EndpointUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Service("machine")
+public class MachineServiceImpl implements IMachineService {
+    static Logger logger = Logger.getLogger(MachineServiceImpl.class.getName());
```

Comment:
```suggestion
    private static Logger logger = Logger.getLogger(MachineServiceImpl.class.getName());
```
This is still default.